layout/templates/wu_story.mustache

```diff
@@ -21,19 +21,19 @@
         
           <div id="fig-caption" class="pie-caption">
             <p>
-            Domestic, livestock, aquaculture, and mining compose the "other" category and togther make up approximately 5 percent of total withdrawals.
+            Domestic, livestock, aquaculture, and mining compose the "other" category and together make up approximately five percent of total withdrawals.
             </p>
           </div>
         </div>
-        
+       
     </div>
-  
+   <p>
+      <i>Each "pie slice" represents a portion of U.S. water withdrawals in 2015.</i>
+   </p>
   </div>
-  
   <div class="story-two">
-  
      <p>
-        Although thermoelectric and irrigation are the most dominant categories nationally, the county breakdown reveals regional variation. Below is a map that shows all categories for each county at one time. While you can still see that irrigation and thermoelectric are the largest, there is a stark contrast between the East, where thermoelectric is dominant, and the West, where irrigation is dominant. See the map below for specific findings:
+        Although thermoelectric and irrigation are the dominant categories nationally, the county breakdown reveals regional variation. In the map below, you can see all categories for each county at once. While you can still see that irrigation and thermoelectric are the largest, there is a striking contrast between the East &mdash; where thermoelectric is dominant &mdash; and the West, where irrigation is dominant.
```

Comment:
For the parentheticals, I'd prefer either em dashes for both east and west or commas for both east and west. Right now it's one of each